Buildings/Fluid/Air/BaseClasses/PartialAirHandlingUnit.mo

```diff
@@ -0,0 +1,231 @@
+within Buildings.Fluid.Air.BaseClasses;
+partial model PartialAirHandlingUnit "Partial AHU model "
+  extends Buildings.Fluid.Interfaces.PartialFourPortInterface(
+    final show_T=true);
+  extends Buildings.Fluid.Air.BaseClasses.EssentialParameter;
+  extends Buildings.Fluid.Actuators.BaseClasses.ValveParameters(
+    final m_flow_nominal=m1_flow_nominal,
+    final rhoStd=Medium1.density_pTX(101325, 273.15+4, Medium1.X_default),
+    final deltaM = deltaM2);
+  extends Buildings.Fluid.Interfaces.FourPortFlowResistanceParameters(
+    final computeFlowResistance1=true,
+    final computeFlowResistance2=true);
+
+  // Cooling coil
+  parameter Boolean waterSideFlowDependent=true
+    "Set to false to make water-side hA independent of mass flow rate"
+    annotation (Dialog(tab="Heat transfer",group="Cooling coil"));
+  parameter Boolean airSideFlowDependent=true
+    "Set to false to make air-side hA independent of mass flow rate"
+    annotation (Dialog(tab="Heat transfer",group="Cooling coil"));
+  parameter Boolean waterSideTemperatureDependent=false
+    "Set to false to make water-side hA independent of temperature"
+    annotation (Dialog(tab="Heat transfer",group="Cooling coil"));
+  parameter Boolean airSideTemperatureDependent=false
+    "Set to false to make air-side hA independent of temperature"
+    annotation (Dialog(tab="Heat transfer",group="Cooling coil"));
+
+  // Initialization of the fan
+  parameter Medium2.AbsolutePressure p_start = Medium2.p_default
+    "Start value of pressure"
+    annotation(Dialog(tab = "Initialization"));
+  parameter Medium2.Temperature T_start=Medium2.T_default
+    "Start value of temperature"
+    annotation(Dialog(tab = "Initialization"));
+  parameter Medium2.MassFraction X_start[Medium2.nX](
+       quantity=Medium2.substanceNames) = Medium2.X_default
+    "Start value of mass fractions m_i/m"
+    annotation (Dialog(tab="Initialization", enable=Medium2.nXi > 0));
+  parameter Medium2.ExtraProperty C_start[Medium2.nC](
+       quantity=Medium2.extraPropertiesNames)=fill(0, Medium2.nC)
+    "Start value of trace substances"
+    annotation (Dialog(tab="Initialization", enable=Medium2.nC > 0));
+  parameter Medium2.ExtraProperty C_nominal[Medium2.nC](
+       quantity=Medium2.extraPropertiesNames) = fill(1E-2, Medium2.nC)
+    "Nominal value of trace substances. (Set to typical order of magnitude.)"
+   annotation (Dialog(tab="Initialization", enable=Medium2.nC > 0));
+  // valve parameters
+  parameter Real l(min=1e-10, max=1) = 0.0001
+    "Valve leakage, l=Kv(y=0)/Kv(y=1)"
+    annotation(Dialog(group="Valve"));
+  parameter Real kFixed(unit="", min=0) = m1_flow_nominal / sqrt(dp1_nominal)
+    "Flow coefficient of fixed resistance that may be in series with valve, k=m_flow/sqrt(dp), with unit=(kg.m)^(1/2)."
```

Comment:
should not use period at the end of comment. So delete `.` at the comment's end.